app/models/google_calendar/event_creator.rb

```diff
@@ -66,7 +69,12 @@ def raise_error_if_invalid(params)
     def add_room_to_event(params, conference_room_id)
       room = ConferenceRoom.find_by(id: conference_room_id)
       raise EventInvalidRoom, "Undefined conference room: #{conference_room_id}" if room.nil?
-      params.merge!(attendees: [{email: room.email}], location: room.title)
+      params[:attendees].unshift(email: room.email, response_status: ACCEPTED_RESPONSE)
```

Comment:
you add attendees element as the first element of the collection, is this done on purpose?